src/library/Uno.Themes.WinUI.Markup/Theme/HyperlinkButton.cs

```diff
@@ -0,0 +1,93 @@
+
+					[ResourceKeyDefinition(typeof(Brush), "SecondaryHyperlinkButtonForeground")]
+					public static ResourceValue<Brush> Secondary => new("SecondaryHyperlinkButtonForeground", true);
+				}
+
+				public static class Background
+				{
+					[ResourceKeyDefinition(typeof(Brush), "HyperlinkButtonBackground")]
+					public static ResourceValue<Brush> Default => new("HyperlinkButtonBackground", true);
+
+					[ResourceKeyDefinition(typeof(Brush), "HyperlinkButtonBackgroundPointerOver")]
+					public static ResourceValue<Brush> PointerOver => new("HyperlinkButtonBackgroundPointerOver", true);
+
+					[ResourceKeyDefinition(typeof(Brush), "HyperlinkButtonBackgroundPressed")]
+					public static ResourceValue<Brush> Pressed => new("HyperlinkButtonBackgroundPressed", true);
+
+					[ResourceKeyDefinition(typeof(Brush), "HyperlinkButtonBackgroundDisabled")]
+					public static ResourceValue<Brush> Disabled => new("HyperlinkButtonBackgroundDisabled", true);
+				}
+
+				public static class BorderBrush
+				{
+					[ResourceKeyDefinition(typeof(Brush), "HyperlinkButtonBorderBrush")]
+					public static ResourceValue<Brush> Default => new("HyperlinkButtonBorderBrush", true);
+
+					[ResourceKeyDefinition(typeof(Brush), "HyperlinkButtonBorderBrushPointerOver")]
+					public static ResourceValue<Brush> PointerOver => new("HyperlinkButtonBorderBrushPointerOver", true);
+
+					[ResourceKeyDefinition(typeof(Brush), "HyperlinkButtonBorderBrushPressed")]
+					public static ResourceValue<Brush> Pressed => new("HyperlinkButtonBorderBrushPressed", true);
+
+					[ResourceKeyDefinition(typeof(Brush), "HyperlinkButtonBorderBrushDisabled")]
+					public static ResourceValue<Brush> Disabled => new("HyperlinkButtonBorderBrushDisabled", true);
+				}
+
+				public static class Typo
+				{
+					[ResourceKeyDefinition(typeof(FontFamily), "HyperlinkButtonFontFamily")]
+					public static ResourceValue<FontFamily> FontFamily => new("HyperlinkButtonFontFamily", true);
+
+					[ResourceKeyDefinition(typeof(double), "HyperlinkButtonFontSize")]
+					public static ResourceValue<double> FontSize => new("HyperlinkButtonFontSize", true);
+				}
+
+				[ResourceKeyDefinition(typeof(double), "HyperlinkButtonBackgroundOpacity")]
+				public static ResourceValue<double> Opacity => new("HyperlinkButtonBackgroundOpacity", true);
+
+				[ResourceKeyDefinition(typeof(double), "HyperlinkButtonBackgroundOpacityPointerOver")]
+				public static ResourceValue<double> OpacityPointerOver => new("HyperlinkButtonBackgroundOpacityPointerOver", true);
+
+				[ResourceKeyDefinition(typeof(double), "HyperlinkButtonBackgroundOpacityPressed")]
+				public static ResourceValue<double> OpacityPressed => new("HyperlinkButtonBackgroundOpacityPressed", true);
+
+				[ResourceKeyDefinition(typeof(double), "HyperlinkButtonBackgroundOpacityDisabled")]
+				public static ResourceValue<double> OpacityDisabled => new("HyperlinkButtonBackgroundOpacityDisabled", true);
```

Comment:
Create an `Opacity` nested class and properties for the different visual state values (with a `Default` for `HyperlinkButtonBackgroundOpacity`)